Sprint_Tests/maryna_boieva.md

```diff
@@ -0,0 +1,17 @@
+Раньше он хорошо разбирался в верстке и работе на передовых технологиях на фронте. После небольшого перерыва в несколько лет и глубокой депрессии плюс прослушивания тяжелой музыки, он слабо разбирается в современных тенденциях и много пьет. Если бы он справился с депрессией после смерти жены и ребенка, у него бы сейчас не стоял вопрос поиска работы с релевантной зарплатой.
+He used to be well versed in markup and advanced frontend technologies. After a few years' break and a deep depression plus listening to hard music, he has a low level of awareness in the current trends and drinks a lot. If he had fought the depression after his wife and child's death, he wouldn't have an issue of finding job with the relevant salary. 
+
+Будь у него цель и адекватные зарплатные ожидания он бы уже работал на удаленке где нибудь в штатах или на Бали с перспективой перебраться на Канарские острова и оформить испанское гражданство. Он с детства мечтал жениться на испанке и рано выйти на пенсию и разводить виноград на побережье Балеарского моря.
+If he had the goal and reasonable salary expectations, he would have alredy worked remotely somewhere in the USA or Bali with the opportunity of moving to the Canary Islands and getting a Spanish citizenship. He has been dreaming since the childhood about marriage with a Spanish woman, early retirement and growing grapes on the Balearic Sea.
+
+Если не знаешь как работает кусок кода, написанный коллегой тебе придется покрыть его консоль логами и проследить всю логическую цепочку. Это трудоемкое занятие. Если бы у нас была документация к этой фиче или хотя бы тикет на реализацию фичи с описанием бизнес логики, мы бы вылили фикс в течении пары часов.
+If you don't know how the part of the code, that had been written by your colleague, works, you will have to cover it with 'console.log's to trace the logic chain. This is a time consuming work. If we had had a documentation for this feature or at least a ticket for feature implementation with the buiseness logic description, we would have pushed a fix within a couple hours.
+
```

Comment:
documentation (uncountable)
the ticket for the feature implementation
a couple of hours